boards/firefly/board.c

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2017 DAI Labor TU Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    boards_firefly
+ * @ingroup     boards
+ * @{
+ *
+ * @file
+ * @brief       Board specific implementations for the Firefly board
+ *
+ * @author      Anon Mall <anon.mall@gt-arc.com>
+ *
+ * @}
+ */
+
+#include "board.h"
+#include "cpu.h"
+#include "fancy_leds.h"
+
+static inline void leds_init(void);
+
+void board_init(void)
+{
+    /* initialize the boards LEDs */
+    leds_init();
```

Comment:
To be consistent with other boards, the gpio init functions should be called after the `cpu_init`